platform/linux-generic/arch/aarch64/odp_atomic.c

```diff
@@ -37,11 +37,18 @@ int odp_atomic_lock_free_u128(odp_atomic_op_t *atomic_op)
 	}
 
 	return 2;
-#else
+#elif defined(__SIZEOF_INT128__)
+	if (__atomic_is_lock_free(16, NULL)) {
+		if (atomic_op) {
+			atomic_op->all_bits = ~((uint32_t)0);
+			atomic_op->op.init  = 0;
```

Comment:
Since API has only three 128bit operations (+ init), it's better to set only those three here.
atomic_op->all_bits = 0;
atomic_op->op.load  = 1;
atomic_op->op.store = 1;
atomic_op->op.cas  = 1;